rfc/src/intro.md

```diff
@@ -8,10 +8,10 @@ integrate feedback from developers and users on future changes to Kani, we decid
 
 You should create an RFC in one of two cases:
 
-1. The change you are proposing would be a "one way door": e.g. a change to the public API, a new feature that would be difficult to modify once released, deprecating a feature, etc.
+1. The change you are proposing would be a "one way door": e.g. a major change to the public API, a new feature that would be difficult to modify once released, etc.
```

Comment:
I think the important parts here are twofold.

1. The public api that the users will use, e.g. UX. Here we probably expect more of just a sketch or a proposal or two of how it _might_ look rather than a definitive answer.
2. Breaking changes to old APIs. Here I think even minor changes are important because they might break existing proofs and if there is any chance that the new features could do so that should be clearly described.

Reply:
@JustusAdam I've been debating about having an RFC to deprecate a feature or breaking changes to old APIs. I am not convinced that today we need a full RFC for that. Once we have our first stable release, this might become more relevant.

Regarding the examples, we probably want the user to copy the template, not past RFCs. Otherwise, it becomes really hard to maintain a source of truth.

Reply:
Oh sorry, I misunderstood the context. I thought this was the section where the changes are described.

Reply:
@JustusAdam would you like me to add that information somewhere else?

Reply:
No I think the contents of the design section cover it well enough